spec/models/merchant_spec.rb

```diff
@@ -4,4 +4,77 @@
   describe 'relationships' do
     it { should have_many :items }
   end
+
+  describe 'enabled_items' do
+    it 'returns items where enabled attribute equals true' do
+      merchant_stephen = Merchant.create!(name: "Stephen's Shady Store")
+      merchant_roger = Merchant.create!(name: "Roger's Fancy Store")
+
+      item_bad_toothpaste = merchant_stephen.items.create!(name: "Bad Toothpaste", description: "The worst toothpaste you can find", unit_price: 4000, enabled: true )
+      item_good_toothpaste = merchant_stephen.items.create!(name: "Super Toothpaste", description: "The best toothpaste you can find", unit_price: 4000, enabled: false )
+
+      item_lamp = merchant_roger.items.create!(name: "Item Lamp", description: "You bet, it's a lamp", unit_price: 4000)
+
+      expect(merchant_stephen.enabled_items).to eq([item_bad_toothpaste])
+    end
+  end
+
+  describe 'disabled_items' do
+    it 'returns items where enabled attribute equals false' do
+      merchant_stephen = Merchant.create!(name: "Stephen's Shady Store")
+      merchant_roger = Merchant.create!(name: "Roger's Fancy Store")
+
+      item_bad_toothpaste = merchant_stephen.items.create!(name: "Bad Toothpaste", description: "The worst toothpaste you can find", unit_price: 4000, enabled: true )
+      item_good_toothpaste = merchant_stephen.items.create!(name: "Super Toothpaste", description: "The best toothpaste you can find", unit_price: 4000, enabled: false )
+
+      item_lamp = merchant_roger.items.create!(name: "Item Lamp", description: "You bet, it's a lamp", unit_price: 4000)
+
+      expect(merchant_stephen.disabled_items).to eq([item_good_toothpaste])
+    end
+  end
+
+  describe 'top_items' do
+    it 'returns the names of the top five items by revenue invoice_item unit price*quantity' do
+
+      customer = Customer.create!(first_name: "Kanye", last_name: "Banjo")
+
+      merchant_stephen = Merchant.create!(name: "Stephen's Shady Store")
+      merchant_roger = Merchant.create!(name: "Roger's Fancy Store")
+
+      item_ski = merchant_stephen.items.create!(name: "Ski", description: "fast skis", unit_price: 10) 
+      item_bike = merchant_stephen.items.create!(name: "Bike", description: "danger bike", unit_price: 10)
+      item_climbing_shoes = merchant_stephen.items.create!(name: "Climbing Shoes", description: "fun shoes", unit_price: 10) 
+      item_snowboard = merchant_stephen.items.create!(name: "Snowboard", description: "slow board", unit_price: 10) 
+      item_rock = merchant_stephen.items.create!(name: "Rock", description: "a good rock", unit_price: 10) 
+
+      item_toothpaste = merchant_stephen.items.create!(name: "Toothpaste", description: "The worst toothpaste you can find", unit_price: 10 )
+      item_bowling_shoes = merchant_stephen.items.create!(name: "Bowling Shoes", description: "not fun shoes", unit_price: 10)
+      item_lamp = merchant_roger.items.create!(name: "Item Lamp", description: "You bet, it's a lamp", unit_price: 10)
```

Comment:
Same issue as I mentioned in the feature test/model

Reply:
Corrected this test as well.